arq/jobs.py

```diff
@@ -247,6 +250,7 @@ def deserialize_result(r: bytes, *, deserializer: Optional[Deserializer] = None)
             result=d['r'],
             start_time=ms_to_datetime(d['st']),
             finish_time=ms_to_datetime(d['ft']),
+            queue_name=d['q'],
```

Comment:
I was slightly concerned that this change will break compatibility if the worker is running an older version of arq if this change goes through then it would cause this to raise a KeyError. Am I right in this thinking and should I change this to `d.get('q')` ?

Reply:
yes I think you're right.

Since it's been so long, I'll fix this myself now.